ExpenseTracker.Core.Tests.Unit/Services/Foundations/Transactions/TransactionServiceTests.Exceptions.Add.cs

```diff
@@ -29,7 +29,9 @@ public async Task ShouldThrowCriticalDependencyExceptionOnAddIfSqlErrorOccurrsAn
                 new FailedTransactionStorageException(sqlException);
 
             var expectedTransactionDependencyException =
-                new TransactionDependencyException(failedTransactionStorageException);
+                new TransactionDependencyException(
+                    message: "Transaction dependency error occurred, contact support."
+                    , innerException: failedTransactionStorageException);
```

Comment:
move the comma up to the end of the previous line

Reply:
do the same for the others

Reply:
Resolved.